config/base/scene/simple_desktop.json

```diff
@@ -810,18 +882,6 @@
         }
       }
     },
-    "csp-wms-overlays": {
-      "maxTextureSize": 1024,
-      "preFetch": 1,
-      "bodies": {
-        "Earth": {
-          "wms": [
-            "https://maps.dwd.de/geoserver/dwd/wms",
-            "https://neo.sci.gsfc.nasa.gov/wms/wms"
-          ]
-        }
-      }
-    },
```

Comment:
Why was this removed?

Reply:
Mostly, because both servers did not work at the time I tested it. Maybe we should test it again. But a larger wall of errors does not bring a good user experience for the default configuration 😉